### PR TITLE
update product requests

### PR DESCRIPTION
**Pull request by andrew-blox**

### CODE REVIEW

File: examples/example-js.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const { SSVScannerCommand } = require('cluster-scanner');
+const { SSVScannerCommand } = require('../dist/tsc/src/commands/SSVScannerCommand');
```
Comment (andrew-blox) [inline examples/example-js.js:1]: I think this is wrong @meshin-blox

**Comment by andrew-blox** [comment]
All things i updated can be found here
https://bloxxx.atlassian.net/browse/IO1-2628

File: examples/example-js.js (hunk 1 above)
Comment (meshin-blox) [reply examples/example-js.js:1]: That's correct, it should use package from node_modules

File: src/commands/SSVScannerCommand.ts [hunk 2]
```diff
@@ -51,7 +55,9 @@ export class SSVScannerCommand {
   }
 
   async getClusterSnapshot(cli: boolean): Promise<IData> {
-    let latestBlockNumber = await Web3Provider.web3(this.params.nodeUrl).eth.getBlockNumber();
+    let latestBlockNumber
+    try { latestBlockNumber = await Web3Provider.web3(this.params.nodeUrl).eth.getBlockNumber(); }
```
Comment (meshin-blox) [inline src/commands/SSVScannerCommand.ts:59]: Format this code block properly

**Review by meshin-blox — CHANGES_REQUESTED** [review]
@andrew-blox please name your branches in future in this format:

```
ab/IO1-2628_cluster_scanner_product_review
```

```
<initials>/<story_number>_<underscored_lowercased_description>
```

File: examples/example-typescript.ts [hunk 3]
```diff
@@ -8,7 +8,12 @@ async function main() {
     operatorIds: [],
   }
   const command = new SSVScannerCommand(params);
-  console.log('Result:', await command.scan());
+  const result = await command.scan()
+  console.log(`{
+    "block": ${result.payload.Block},
+    "cluster snapshot": ${JSON.stringify(result.cluster, null, "\t")},
+    "cluster": [${Object.values(result.cluster)}]
+}`);
```
[the remark this answers is not in the document]
Comment (andrew-blox) [reply examples/example-typescript.ts:16]: updated

File: src/cli-shared.ts [hunk 4]
```diff
@@ -42,7 +42,7 @@ export default async function main(): Promise<any> {
   });
 
   
-  const messageText = `SSV Scanner v${pkg.version}`;
```
[the remark this answers is not in the document]
Comment (andrew-blox) [reply src/cli-shared.ts:45]: updated

File: src/cli-shared.ts [hunk 5]
```diff
@@ -65,8 +65,13 @@ export default async function main(): Promise<any> {
...
     const result = await command.execute();
     console.table(result.payload);
-    console.log('\CLuster snapshot:');
+    console.log('\Cluster snapshot:');
     console.table(result.cluster);
+console.log(`"raw": {
+  "block": ${result.payload.Block},
+  "cluster snapshot": ${JSON.stringify(result.cluster, null, "\t")},
+  "cluster": [${Object.values(result.cluster)}]
+}`);
```
[the remark this answers is not in the document]
Comment (andrew-blox) [reply src/cli-shared.ts:74]: udpatec

File: src/commands/SSVScannerCommand.ts [hunk 6]
```diff
@@ -36,6 +36,10 @@ export class SSVScannerCommand {
     if (!Array.isArray(params_.operatorIds) || !this.isValidOperatorIds(params_.operatorIds.length)) throw Error('Operator ids list is not valid');
     if (!params_.ownerAddress) throw Error('Cluster owner address is required');
     this.params = params_;
+    if (params_.contractAddress.length !== 42) throw Error('Invalid contract address length.');
+    if (params_.contractAddress.slice(0, 2) !== '0x') throw Error('Invalid contract address.');
```
[the remark this answers is not in the document]
Comment (andrew-blox) [reply src/commands/SSVScannerCommand.ts:40]: udpated

File: src/commands/SSVScannerCommand.ts [hunk 7]
```diff
@@ -36,6 +36,10 @@ export class SSVScannerCommand {
...
+    if (params_.ownerAddress.length !== 42) throw Error('Invalid owner address length.');
+    if (params_.ownerAddress.slice(0, 2) !== '0x') throw Error('Invalid owner address.');
```
[the remark this answers is not in the document]
Comment (andrew-blox) [reply src/commands/SSVScannerCommand.ts:42]: updated

File: src/commands/SSVScannerCommand.ts [hunk 8]
```diff
@@ -51,7 +55,9 @@ export class SSVScannerCommand {
   }
 
   async getClusterSnapshot(cli: boolean): Promise<IData> {
-    let latestBlockNumber = await Web3Provider.web3(this.params.nodeUrl).eth.getBlockNumber();
+    let latestBlockNumber
```
[the remark this answers is not in the document]
Comment (andrew-blox) [reply src/commands/SSVScannerCommand.ts:58]: updated

File: src/commands/SSVScannerCommand.ts (hunk 2 above)
Comment (andrew-blox) [reply src/commands/SSVScannerCommand.ts:59]: updated

**Review by meshin-blox — APPROVED** [review]

**Pull request merged**